bicep/main.bicep

```diff
@@ -1207,6 +1207,9 @@ var aks_addons1 = ingressApplicationGateway ? union(aks_addons, deployAppGw ? {
 @description('Sets the private dns zone id if provided')
 var aksPrivateDnsZone = privateClusterDnsMethod=='privateDnsZone' ? (!empty(dnsApiPrivateZoneId) ? dnsApiPrivateZoneId : 'system') : privateClusterDnsMethod
 output aksPrivateDnsZone string = aksPrivateDnsZone
+output privateFQDN string = !empty(privateClusterDnsMethod) ? aks.properties.privateFQDN : ''
```

Comment:
I think the condition will need to be `enablePrivateCluster && privateClusterDnsMethod != 'none'`

param privateClusterDnsMethod is never empty, the conditional nature and variety of dns options for private clusters mean that we need a more complex condition.